tests/test_images.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+import os
+import shutil
+import tempfile
+import unittest
+
+import pytz
+
+from server import images
+
+
+class ImagesTest(unittest.TestCase):
+
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+        self.indexer = images.Indexer(self.temp_dir)
+
+    def tearDown(self):
+        shutil.rmtree(self.temp_dir)
+
+    def test_indexes_image_files(self):
+        open(os.path.join(self.temp_dir, '2017-04-01T1851Z.jpg'), 'a').close()
```

Comment:
Oh, this is kind of confusing. `open` creates a file and opens a file handle to it, then `close` closes the handle, but the file is still on the disk. The image indexer just looks at filenames but never actually opens the files so it doesn't need handles to the files.

Refactored into a helper function to try and clarify.